deploy/docs/collecting-kubernetes-events.md

```diff
@@ -63,7 +54,54 @@ Example Kubernetes event:
 
 ## Configuration
 
-To configure event collection, see the following sections of the [values.yaml][values_yaml] file, depending on the provider used:
+Event collection configuration can be found under the `sumologic.events` key of the [values.yaml][values_yaml] file.
+
+### Setting source name and category
+
+It's possible to customize the [source name][source_name] and [category][source_category] for events:
+
+```yaml
+sumologic:
+  events:
+    sourceName: myEventSource
+    sourceCategory: myCustomSourceCategory
+```
+
+### Customizing persistence
+
+By default, the event collector provisions and uses a Kubernetes PersistentVolume to persist some information over service restarts.
+In particular, the collector remembers the most recently processed Event this way, thus avoiding having to reprocess past Events
+after restart. The Persistent Volume is also used to buffer Event data if the remote destination is inaccessible.
+
+Persistence can be customized via the `sumologic.events.persistence` section:
+
+```yaml
+sumologic:
+  events:
+    persistence:
+      size: 10Gi
+      path: /var/lib/storage/events
+      accessMode: ReadWrite
```

Comment:
Shouldn't this go to `otelevents` configuration?
It sounds like more advanced and technical configuration